src/test/java/systemtests/HelpCommandSystemTest.java

```diff
@@ -31,6 +36,13 @@
 
     @Test
     public void openHelpWindow() {
+
```

Comment:
Add a assertSuccess to check if help window open without the need to login

Reply:
Resolved inside [151829b](https://github.com/CS2103-AY1819S1-T16-3/main/pull/52/commits/151829b89501efcafd8b314dbf6032764fb69cb2)